packages/babel-helper-module-imports/README.md

```diff
@@ -1,10 +1,60 @@
 # babel-helper-module-imports
 
+## Installation
+
+```sh
+npm install babel-helper-module-imports --save
+```
+
 ## Usage
 
-### Adding a named import
+### `import "source"`
 
+```js
+import { addSideEffect } from "babel-helper-module-imports";
+addSideEffect(path, 'source');
 ```
+
+### `import { named } from "source"`
+
+```js
+import { addNamed } from "babel-helper-module-imports";
+addNamed(path, 'named', 'source');
+```
+
+### `import { named as _hintedName } from "source"`
+
+```js
+import { addNamed } from "babel-helper-module-imports";
+addNamed(path, 'named', 'source', { nameHint: "hintedName" });
```

Comment:
@hzoo would be nice to add some docs about what `nameHint` is

is it just a local specifier? if so the the example input doesnt match example output here (`hintedName` vs `_hintedName`)

Reply:
for sure, its just because or renaming of a causes _a